pages/builds/sharing_state.md.erb

```diff
@@ -0,0 +1,50 @@
+# Sharing State Across Steps
+
+Sharing state across the steps of your build removes repetion and speeds up your build time. Passing data like npm packages and docker containers between steps can help keep your steps single-purpose (and cuts down the time each step takes and how much processing it has to do).   
+
+There are two ways to share state across steps in a build: artifacts and meta-data. Both of these data stores are accessible across your organisation and pipelines.  
+
+<%= toc %>
+
+## Build Artifacts
+
+Artifacts are any files that are generated during a build step. Buildkite can host these files for you in the Buildkite artifact store. 
```

Comment:
I view artifacts as products of a build that are useful to humans, or an input for a later build step.

Reply:
Artifacts can be anything though, right? And we recommend that people only keep useful things/things they want to use later? I'd like to leave the first sentence there, and then add a recommendation about use. Yeah/nah?

Reply:
Yeah, thats ok.